verifier/helpers.go

```diff
@@ -55,7 +55,7 @@ func loadX509TrustStores(ctx context.Context, scheme signature.SigningScheme, po
 
 		storeType, name, found := strings.Cut(trustStore, ":")
 		if !found {
-			return nil, fmt.Errorf("trust policy statement %q is missing separator in trust store value %q", policy.Name, trustStore)
+			return nil, fmt.Errorf("trust policy statement %q is missing separator in trust store value %q. Format <TrustStoreType>:<TrustStoreName> is required", policy.Name, trustStore)
```

Comment:
nit: Should we reword it to:
`trust policy statement %q is missing separator in trust store value %q. The required format is <TrustStoreType>:<TrustStoreName>`